test/project/example.gdextension

```diff
@@ -5,14 +5,22 @@ compatibility_minimum = "4.1"
 
 [libraries]
 
+ios.debug = "res://bin/libgdexample.ios.template_debug.xcframework"
+ios.release = "res://bin/libgdexample.ios.template_release.xcframework"
```

Comment:
The ordering of the libraries seems a bit arbitrary.
Currently it seems to be desktop platforms first, and then mobile/web, so I'd suggest moving `ios` next to `android`.

Reply:
Moved iOS libs next to Android and added a comment for `p_dummy`.